pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -64,6 +64,17 @@ var targetOnlineRestoreSpanSize = settings.RegisterByteSizeSetting(
 	16<<30,
 )
 
+var maxFileCount = settings.RegisterIntSetting(
+	settings.SystemOnly,
+	"backup.restore_span.max_file_count",
+	"the max number of backup files an extending restore span may contain",
+	defaultMaxFileCount,
+	settings.PositiveInt,
+	settings.WithPublic,
```

Comment:
Do we definitely need this to be public? target_size isn't public

Reply:
good q: If we make it public, the user (in theory) could speed up their restore on their own, without support intervention. 

I don't feel strongly though.

Reply:
they could also crash their cluster, as we discovered.

Reply:
cool, i'll make this private.